tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "strictBindCallApply": true,
     "strictNullChecks": true,
     "strictPropertyInitialization": true,
+    "esModuleInterop": true,
```

Comment:
I don't recommend setting this flag.  It completely changes how all the `import` statements are transpiled and can result in unintuitive import behavior that causes runtime bugs.  I'm guessing you probably set this flag so you could import the `bs58` package like this:

```javascript
import bs58 from "bs58"
```

I suggest removing the flag and instead importing the library using this syntax:

```javascript
import * as bs58 from "bs58"
```

Reply:
Yup, that's what I wanted. Thanks!